tests/elm-doc-test.json

```diff
@@ -1,4 +0,0 @@
-{
-    "root": "../src",
-    "tests": ["WeakCss"]
-}
```

Comment:
this means we loose verification of docs examples? @Janiczek do you know if elm-verify-examples is ready for 0.19?

Reply:
It has 6 days old v3.0.0 - I haven't tried it but would assume that's 0.19-ready. I'm not next to PC so I can't try - you can try and use it!

Reply:
Yes I have dropped the Support for Elm doc tests however they can be easily adres obce they are 0.19 compatible

Reply:
I have tried elm-verify-examples with 0.19 and it is working like a charm, so I rewrote the doc-tests to have the new syntax and using elm-verify-examples now. And it is working well :)